ui/StatusQ/src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -41,6 +41,9 @@ Flickable {
     readonly property int availableWidth: width - leftPadding - rightPadding
     readonly property int availableHeight: height - topPadding - bottomPadding
 
+    implicitWidth: contentWidth + leftPadding + rightPadding
+    implicitHeight: contentHeight + topPadding + bottomPadding
+
```

Comment:
most probably this branch is outdated, as this is already in master: https://github.com/status-im/status-desktop/pull/9384

Reply:
Good then, removed this part!
thanks 🙂